packages/fetch-cached-dns/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  extends: [require.resolve("@vercel/style-guide/eslint")],
+  rules: {
+    "no-bitwise": "off",
+    "no-param-reassign": "off",
+  },
+};
```

Comment:
Can we have a single eslint at the root of the monorepo instead of configuring each pacakge?

Reply:
I wanted this too but I don't think Turbo then caches the results of lint/format

Reply:
@jaredpalmer Is that true? Seems like Turbo should be able to handle a single setting in the root of the monorepo

Reply:
I think we'll need to move this way in future anyway @styfle, as TypeScript support in ESLint struggles in large repos.

We hit that in API, and @dglsparsons discovered that the team behind that plugin also recommend per-package configs for now.

That being said, I'd prefer a single root config too... It's much easier to maintain.

Reply:
It should be possible to have a small amount of config per package that simply extends a base config though? Can even have it in the package.json. 
Or are we saying even that is too much.

Reply:
The point of the style guide is consistency. So I personally feel that turning off different rules in each repo works against that idea - unless it's temporary, during migration.

I'm also concerned that it's a slippery slope. Engineers will see a bunch of disabled rules, and then add to that.

Of course, the exception would be where specific tooling is used and additional plugins/rules are needed to support that.

Reply:
Due note that these packages are going to be rewritten in TS soon; but for now I'm trying to iteratively stabilize the existing modules before doing so. I'm going to move forward with the disabled rules, but have full intention to 100% follow the style-guide during the TS rewrite.